Description: Adjusts the version of the com.sun.mail:mailapi dependency
 to help maven-debian-helper identifying as a reactor dependency.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/parent-distrib/pom.xml
+++ b/parent-distrib/pom.xml
@@ -142,7 +142,7 @@
 	<dependency>
 	    <groupId>com.sun.mail</groupId>
 	    <artifactId>mailapi</artifactId>
-	    <version>${mail.version}</version>
+	    <version>${project.version}</version>
 	</dependency>
     </dependencies>
 </project>
